addons/mail/static/src/discuss/core/common/suggestion_service_patch.js

```diff
@@ -42,7 +42,7 @@ const suggestionServicePatch = {
                 .filter((persona) => persona.type === "partner");
```

Comment:
This is not adapted, make sure to double check all usage of of `persona`/`type`/`"partner"`/`"guest"`